colin/checks/labels/summary.py

```diff
@@ -0,0 +1,14 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class SummaryCheck(LabelCheck):
+
+    def __init__(self):
+        super().__init__(name="summary_label_required",
+                         message="Label 'summary' has to be specified.",
+                         description="A short description of the image.",
+                         reference_url="http://url-to-upstream-documentation/summary",
```

Comment:
Do we have some real url for all of these checks?

Reply:
What about this one https://fedoraproject.org/wiki/Container:Guidelines#LABELS ?

Reply:
If the fedora-specific document is ok, then we can add this to all of the `label` checks.

Reply:
definitely good enough (for now)